pkg/userinfo/roleRef.go

```diff
@@ -62,7 +63,7 @@ func getRoleRefByRoleBindings(roleBindings []*rbacv1.RoleBinding, userInfo authe
 func getRoleRefByClusterRoleBindings(clusterroleBindings []*rbacv1.ClusterRoleBinding, userInfo authenticationv1.UserInfo) (clusterRoles []string) {
 	for _, clusterRoleBinding := range clusterroleBindings {
 		for _, subject := range clusterRoleBinding.Subjects {
-			if matchSubjectsMap(subject, userInfo) {
+			if matchSubjectsMap(subject, userInfo, clusterRoleBinding.Namespace) {
```

Comment:
The namespace is empty for clusterrolebinding, you still need to set it from `subject.namespace`.

Reply:
Got it, updating it

Reply:
Done